addons/website_jitsi/models/chat_room.py

```diff
@@ -53,5 +54,14 @@ def _compute_is_full(self):
             else:
                 room.is_full = room.participant_count >= int(room.max_capacity)
 
+    def _compute_jitsi_server_domain(self):
+        jitsi_server_domain = (
+            self.env['ir.config_parameter'].sudo().get_param('website_jitsi.jitsi_server_domain')
```

Comment:
The "get_param" method has a default mechanism (second parameter of the function)